src/components/Randombeer.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { useState, useEffect } from 'react';
+import Header from './Header';
+import axios from 'axios';
+
+function Randombeer() {
+  const [random, setRandom] = useState({});
+  const [randomBeerDetail, setRandomBeerDetail] = useState({});
```

Comment:
There's no need to have these two useState here. You should have only one for now